examples/otel-operator/app.js

```diff
@@ -19,24 +19,31 @@
 
 // A simple Express app with a `GET /ping` endpoint.
 
+const bunyan = require('bunyan');
 const express = require('express');
+const {metrics} = require('@opentelemetry/api');
+
+const log = bunyan.createLogger({
+    name: 'myapp',
+    serializers: bunyan.stdSerializers,
+});
+
+// A silly counter to demonstrate using custom OTel metrics.
+const meter = metrics.getMeter('myapp');
+const counter = meter.createCounter('num_pings');
```

Comment:
reviewer note: Make the example app more interesting by showing custom metrics and logging usage to see if telemetry for those work.